tests/scripts/kubeadm-install.sh

```diff
@@ -74,3 +74,10 @@ sudo apt-get install -y kubelet="${KUBE_INSTALL_VERSION}"  && sudo apt-get insta
 wget "https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl"
 chmod +x kubectl
 sudo cp kubectl /usr/local/bin
+
+sudo apt-get update
```

Comment:
is there a reason we need to call apt-get update again? we just called it on line 66